src/framework/parsers/glb-parser.js

```diff
@@ -2077,13 +2085,15 @@ const createImages = (gltf, bufferViews, urlBase, registry, options) => {
                 if (gltfMaterial.hasOwnProperty('pbrMetallicRoughness')) {
                     const pbrData = gltfMaterial.pbrMetallicRoughness;
                     if (pbrData.hasOwnProperty('baseColorTexture')) {
-                        set.add(pbrData.baseColorTexture.index);
+                        const textureIndex = getImageIndexFromTextureIndex(gltf, pbrData.baseColorTexture.index);
```

Comment:
I'd probably remove `getImageIndexFromTextureIndex()` function as it isn't adding much.